game/ato/flightplans/waypointbuilder.py

```diff
@@ -494,59 +501,105 @@ def escort(
         )
 
     @staticmethod
-    def stopover(stopover: ControlPoint, name: str = "STOPOVER") -> FlightWaypoint:
-        """Creates a stopover waypoint.
+    def land_refuel(
+        control_point: ControlPoint, is_destination: bool = False
```

Comment:
afaict this is only ever called once with `is_destination=True`. Remove the arg and prune the unused branch?

Reply:
(looking more closely I think this function only exists because of a bug, so probably remove it entirely)

Reply:
It is used to give the waypoint the correct name "LANDING". Due to the DCS bug that i mentioned in the comment we have to add another LandingReFuAr Waypoint. Otherwise the AI wont land.
For me this is fine, everything is described. there are many comments.
Why more effort...

Reply:
> there are many comments. Why more effort...

Because afaict this is a bug. This has exactly one caller, used to create the _arrival_ waypoint for airlift flights. The comment on the caller is wrong:

```python
# The AI Needs another landing&refuel point to actually fly back to the original
# base. Otherwise the Cargo drop will be the new Landing Waypoint and the
# AI will end its mission there instead of flying back.
```

`arrival` should not be using `LAND_REFUEL`, which will cause it to land, refuel, take off, and return to some arbitrary airfield. It is supposed to land there and stay there, which is what `LANDING_POINT` does.